### ISSUE TITLE
@quasar/extras@1.16.10 breaks fontawesome-v6

### ISSUE THREAD

**Issue by anteqkois** [quasarframework/quasar#17052]
### What happened?

My build starts fails at production. I found that here are installing @quasar/extras@1.16.10. I checked locally and with fixed version @quasar/extras@1.16.9 all works fine, but when I switch to @quasar/extras@1.16.10, this error occurs.

![image](https://github.com/quasarframework/quasar/assets/73034007/895cc2cc-9eba-485b-9923-b703c28d5c97)

### What did you expect to happen?

Succesfully build

### Reproduction URL

i can't provide repo etc. it's private (company)

### How to reproduce?

1. add fontawesome icons
2. install @quasar/extras@1.16.10
3. `quasar build`

### Flavour

Quasar CLI with Webpack (@quasar/cli | @quasar/app-webpack)

### Areas

Plugins (quasar), Quasar Extras (@quasar/extras), App Extension API

### Relevant log output

```Shell
App •  ERROR  •  UI  in ./node_modules/.pnpm/@quasar+extras@1.16.10/node_modules/@quasar/extras/fontawesome-v6/fontawesome-v6.css

Module not found: Can't resolve imported dependency "../webfonts/fa-brands-400.woff2"

 App •  ERROR  •  UI  in ./node_modules/.pnpm/@quasar+extras@1.16.10/node_modules/@quasar/extras/fontawesome-v6/fontawesome-v6.css

Module not found: Can't resolve imported dependency "../webfonts/fa-brands-400.ttf"


 App •  COMPILATION FAILED  • Please check the log above for details.
```

**Comment by rstoenescu** [comment]
Thanks for reporting!
Fix will be available shortly in q/extras 1.16.11

**Comment by rstoenescu** [comment]
Released it.
